app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        vaccinated = visitor.get("vaccine", None)
+        mask = visitor.get("wearing_a_mask", None)
```

Comment:
```suggestion
        vaccinated = visitor.get("vaccine")
        mask = visitor.get("wearing_a_mask")
```

Get returns None by default

Reply:
I know,but in vscode i have autoinput and i forgot to check in this place)))